classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @student = []
```

Comment:
- [ ] Same issue as above
```suggestion
    @students = []
```